### PR TITLE
Debian version upgrade

### PR DESCRIPTION
**Pull request by juha-h**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+baresip (3.6.1) UNRELEASED; urgency=medium
```
Comment (sreimers) [inline debian/changelog:1]: The current version in main is 3.6.0

**Comment by juha-h** [comment]
Sebastian Reimers writes:

> > @@ -1,3 +1,9 @@
> +baresip (3.6.1) UNRELEASED; urgency=medium
> 
> The current version in main is 3.6.0

I know, but since tag 3.6.0 there has been many commits to main.  Are
you planning to re-tag 3.6.0?

**Comment by sreimers** [comment]
I'm not sure if we need a extra release for baresip. I just released re v3.6.1 with the relevant fix. We should not bump debian changelog version independent from official version I think. We always increase version numbers only on releases. So fixing the debian version number in main should be fine.

**Comment by juha-h** [comment]
Sebastian Reimers writes:

> I'm not sure if we need a extra release for baresip. I just released
> re v3.6.1 with the relevant fix. We should not bump debian changelog
> version independent from official version I think. We always increase
> version numbers only on releases. So fixing the debian version number
> in main should be fine.

So this didn't fix any bug in 3.6.0?

https://github.com/baresip/baresip/commit/8c2f58336bb585ca11e3fe36c0501b0f04c47701

**Comment by sreimers** [comment]
> So this didn't fix any bug in 3.6.0? [8c2f583](https://github.com/baresip/baresip/commit/8c2f58336bb585ca11e3fe36c0501b0f04c47701)

Not sure, maybe @cspiel1 can decide if this is critical enough for a extra release.

**Comment by juha-h** [comment]
> > So this didn't fix any bug in 3.6.0? [8c2f583](https://github.com/baresip/baresip/commit/8c2f58336bb585ca11e3fe36c0501b0f04c47701)
> 
> Not sure, maybe @cspiel1 can decide if this is critical enough for a extra release.

If not, then I can edit the PR so that Debian version is 3.6.0.  Current one 3.5.0 is not correct.

**Comment by cspiel1** [comment]
It was not a critical bug. It fixes only that too many UA events are thrown. At least it is not worth to force a creation of a release branch `3.6.x`.

**Comment by alfredh** [comment]
Thanks!

Unrelated: it looks like Homebrew has deprecated macOS 11, do we need to build for both macOS 11 and 12 ?

**Pull request merged**